build/vfs.js

```diff
@@ -55,30 +75,38 @@ export default function env(inst) {
         buffer_ptr,
         amount,
       );
-      Deno.seekSync(rid, offset, Deno.SeekMode.Start);
-      return Deno.writeSync(rid, buffer);
+      const file = getOpenFile(rid);
+      file.seekSync(offset, Deno.SeekMode.Start);
+      return file.writeSync(buffer);
     },
     // Truncate the given file
     js_truncate: (rid, size) => {
-      Deno.ftruncateSync(rid, size);
+      const file = getOpenFile(rid);
+      file.truncateSync(size);
     },
     // Sync file data to disk
     js_sync: (rid) => {
-      Deno.fdatasyncSync(rid);
+      const file = getOpenFile(rid);
+      file.syncDataSync();
```

Comment:
@dyedgreen I don't know of a good way to make this method support Deno versions before v1.44.

* We could make it a no-op like the lock/unlock method. Is that acceptable or would that break things?
* Or we could bump the minimum supported Deno version to v1.44 after this lands.

What do you think?

Reply:
@dyedgreen Let me know when you get a chance to look at this. I'm thinking we may need to bump the minimum supported Deno version to get access to the new file flushing APIs. What are your thoughts?